cinema/views.py

```diff
@@ -0,0 +1,45 @@
+from django.http import HttpResponse, JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework.parsers import JSONParser
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@csrf_exempt
```

Comment:
You can use rest_framework's `@api_view` instead.

Reply:
Ok, thanks! It was science experiment.